src/main/java/com/celements/search/web/WebSearchScriptService.java

```diff
@@ -80,4 +81,13 @@ public LuceneSearchResult webSearch(String searchTerm, DocumentReference configD
     return ret;
   }
 
+  public List<String> getAvailablePackages(DocumentReference configDocRef) {
+    List<String> ret = new ArrayList<>();
+    Set<WebSearchPackage> searchPackages = searchService.getAvailablePackages(configDocRef);
```

Comment:
NPE possible since `getAvailablePackages(@NotNull DocumentReference configDocRef)`. Instead of a null check in the ScriptService better change `IWebSearchService` to accept Nullable, it will then only return the default packages.